forums/forums-war/src/main/webapp/forums/jsp/viewMessage.jsp

```diff
@@ -65,6 +67,8 @@
     int currentMessageId = -1;
     boolean scrollToMessage = false;
     boolean displayAllMessages = true;
+    boolean isAccessGuest = fsc.isAccessGuest();
```

Comment:
If the instruction `<jsp:useBean id="isAccessGuest" type="java.lang.Boolean"/>` is used just after line 43 of this source file, then JSTL variable `isAccessGuest` is accessible into scriptlet context.
So `boolean isAccessGuest = fsc.isAccessGuest();` can be removed and `isAccessGuest` method of ForumSessionController should also be removed.